blocks/address/src/address-card.tsx

```diff
@@ -0,0 +1,340 @@
+          position: "relative",
+          background: palette.gray[10],
+          borderLeft: `1px solid ${palette.gray[20]}`,
+          width: 500,
+          [theme.breakpoints.down("md")]: {
+            width: 1,
+            height: 300,
+          },
+        })}
+      >
+        {mapUrl ? (
+          <Box
+            sx={{
+              width: 1,
+              height: 1,
+              background: `url(${mapUrl}) no-repeat`,
+              backgroundSize: "cover",
+            }}
+          >
+            <Fade in={hovered}>
+              <Stack
+                sx={{
+                  display: "inline-flex",
+                  position: "absolute",
+                  top: 13,
+                  left: 13,
+                }}
+              >
+                <Button
+                  onClick={incrementZoomLevel}
+                  disabled={!incrementZoomLevel}
+                  variant="tertiary"
+                  sx={{
+                    minWidth: "unset",
+                    minHeight: "unset",
+                    padding: 0.5,
+                    borderBottomLeftRadius: 0,
+                    borderBottomRightRadius: 0,
+                    borderBottomWidth: 0,
+                  }}
+                >
+                  <FontAwesomeIcon icon={faPlus} />
+                </Button>
+                <Button
+                  onClick={decrementZoomLevel}
+                  disabled={!decrementZoomLevel}
+                  variant="tertiary"
+                  sx={{
+                    minWidth: "unset",
+                    minHeight: "unset",
+                    padding: 0.5,
+                    borderTopLeftRadius: 0,
+                    borderTopRightRadius: 0,
+                  }}
+                >
+                  <FontAwesomeIcon icon={faMinus} />
+                </Button>
+              </Stack>
+            </Fade>
+
+            <Typography
+              sx={{
+                position: "absolute",
+                bottom: 13,
+                right: 13,
+                color: ({ palette }) => palette.common.black,
+                opacity: 0.5,
+                fontSize: 9,
+                lineHeight: "11px",
+              }}
+            >
+              ©{" "}
+              <Link
+                href="https://www.mapbox.com/about/maps/"
+                target="_blank"
+                sx={{ color: "inherit", textDecoration: "none" }}
+              >
+                Mapbox
+              </Link>{" "}
+              ©{" "}
+              <Link
+                href="https://www.openstreetmap.org/copyright"
+                target="_blank"
+                sx={{ color: "inherit", textDecoration: "none" }}
+              >
+                OpenStreetMap
+              </Link>
+            </Typography>
+          </Box>
+        ) : (
+          <CircularProgress sx={{ color: ({ palette }) => palette.gray[40] }} />
+        )}
+
+        <Fade in={hovered}>
```

Comment:
This should not appear in `readonly`

Reply:
Hid it in readonly mode in 38c4a19ea9d64daea84706f708acccffd3e7ebc9